docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -388,6 +388,8 @@ See also `Column Properties`_.
 
 ``ALTER TABLE ... DROP COLUMN`` is only allowed if not part of a primary key.
 
+``ALTER TABLE ... SET PROPERTIES`` is unsupported.
```

Comment:
Plain old English would be:
   ALTER TABLE ... SET PROPERITES is not supported.

Unless we are saying "thou are cast into unsupported"